src/main/scala/nl/knaw/dans/easy/license/CommandLineOptions.scala

```diff
@@ -15,35 +15,88 @@
  */
 package nl.knaw.dans.easy.license
 
-import java.io.{File, PrintWriter}
-import java.net.URL
+import java.io.File
+import javax.naming.Context
+import javax.naming.ldap.InitialLdapContext
 
-import org.rogach.scallop.ScallopConf
+import com.yourmediashelf.fedora.client.FedoraCredentials
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.rogach.scallop._
+import org.slf4j.LoggerFactory
 
-class CommandLineOptions (args: Array[String]) extends ScallopConf(args) {
-  printedName = "easy-license-creator";
-  val __________ = " " * printedName.size
+class CommandLineOptions(args: Array[String]) extends ScallopConf(args) {
+  
+  appendDefaultToDescription = true
+  editBuilder(_.setHelpWidth(110))
+
+  val fileMayNotExist = singleArgConverter(new File(_))
+
+  printedName = "easy-license-creator"
 
   version(s"$printedName v${Version()}")
   banner(s"""
-           |<Replace with one sentence describing the main task of this module>
+           |Create a license for the given datasetID and depositorID. The latter is optional as it can also
+           | be retrieved from the datasetID. The license will be saved at the indicated location.
```

Comment:
Good catch. That was legacy from a former era when we had `depositorID` as command line parameter to avoid a number of Fedora queries. This however did not make sense because you would have to check whether the `depositorID` is valid and with that still query the `AMD`. So we decided to deprecate this parameter. Forgot to delete that part of the test. See 1be4b70ff33b64b319fe0ca8ea95294f29e04ef0